packages/import/src/index.ts

```diff
@@ -360,9 +198,246 @@ function visitFile(
       }
     }
   }
-  return visitedFiles.get(filePath);
+  return visitedFiles.get(filePath)!;
```

Comment:
Note the `!` at the end - we know that there is an entry for `filePath` because we put it in the map earlier.